.semaphore/semaphore.yml

```diff
@@ -1,21 +1,38 @@
-version: v1.0
-name: Go
-agent:
-  machine:
-    type: e1-standard-2
-    os_image: ubuntu2004
-blocks:
-  - name: Test
-    task:
-      jobs:
-        - name: go test
-          commands:
-            - sem-version go 1.17
-            - export GO111MODULE=on
-            - export GOPATH=~/go
-            - 'export PATH=/home/semaphore/go/bin:$PATH'
-            - checkout
-            - go version
-            - go get ./...
-            - go test ./...
-            - go build -v .
+version: v1.0
+name: Go
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu2004
+blocks:
+  - name: Style Check
+    task:
+      jobs:
+        - name: fmt
+          commands:
+            - sem-version go 1.17
+            - checkout
+            # Recommended way to install golang-ci
+            # https://golangci-lint.run/usage/install/#other-ci.
+            - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.42.1
```

Comment:
One thing I always do with these kinds of `curl | sh` scripts is add them to the repository so that the script contents are version controlled. Otherwise you never know what's on the other side of that remote call.

Reply:
I'd be happy to vendor the script, although, this particular script I also tend to download in some pipelines

Reply:
Yep, fully agree. Will vendor. :+1: